test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# RUN FROM ROOT DIRECTORY OF REPOSITORY
+
+# put input data's root folder here
+juicer_data=""
```

Comment:
Recommend using "$1" for a primitive argument parsing.  Then can call `test.sh juicer/data/path` and not have to modify the script (triggering a repo diff).

Reply:
Agreed, will tackle this at a future point.